src/pages/csr.tsx

```diff
@@ -115,18 +116,19 @@ const CSRStats: NextPage<CSRStatsProps> = ({ csrStats, chainConfig }) => {
 export default CSRStats
 
 export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
+  if (CHAIN_ID !== CHAIN_IDS.CANTO) {
+    return
+  }
   const apolloClient = initializeApollo({ ctx })
 
-  const {
-    data: { csrStats, chainConfig },
-  } = await apolloClient.query({
+  const { data } = await apolloClient.query({
     query: GET_CSR_STATS,
   })
 
   return addApolloState(apolloClient, {
     props: {
-      csrStats,
-      chainConfig,
+      csrStats: data.csrStats || null,
```

Comment:
```suggestion
      csrStats: data?.csrStats,
```

Reply:
looooong thread why you can't pass undefined ti props in server side next.js 

https://github.com/vercel/next.js/discussions/11209

I tried with nullable operator,  but then got the above error, and changed fallback to `null`